### ISSUE TITLE
Development plan in v.0.6.0

### ISSUE THREAD

**Issue by ShigekiKarita** [espnet/espnet#1164]
feel free to edit this and request new things
- [x] speech translation and ASR multi task learning #1139 
- [ ] unify the beam search interface based on batch processing
  - [ ] batch processing for LM scoring in beam search
- [ ] transformer fast inference
- [ ] fix multiple GPU issues (distributed #688, non chainer trainer, data loader)
- [ ] use of transformer LM
- [ ] LMInterface in Chainer #1111 
- [ ] OptimizerFactoryInterface #943 
- [ ] Frondend in Transformer #890
- [ ] Remove `--nowarn` in travis-sphinx build
- [ ] officially support transfer learning
  - [ ] release more pre-trained models as a seed model for transfer learning
  - [ ] use BERT or other existing pre-trained models
  - Note that speech translation and RNN transducer already use them
- [ ] fix negative CTC score issues (only happens in ami and chime5)
- [ ] multiple encoders
- [ ] merge ASR-TTS #818 
  - [ ] extend it with Transformer
- [ ] check initialization effects
